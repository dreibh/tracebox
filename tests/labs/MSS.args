-q -p IP/TCP/MSS
