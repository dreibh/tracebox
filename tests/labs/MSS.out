tracebox to 1.2.3.4 (1.2.3.4): 64 hops max
1: 1.1.1.1 0ms TCP::CheckSum TCPOptionMaxSegSize::MaxSegSize 
2: 1.2.3.4 0ms TCP::CheckSum IP::TTL IP::CheckSum TCPOptionMaxSegSize::MaxSegSize 
